package.json

```diff
@@ -86,10 +86,10 @@
     "chai-as-promised": "^5.2.0",
     "chai-subset": "^1.6.0",
     "documentation": "^8.1.2",
-    "eslint": "^4.17.0",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-mocha": "^5.3.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-mocha": "^6.2.2",
```

Comment:
may we consider using https://greenkeeper.io/ to keep us more peace?

Reply:
don't know.
create an feature issue